content/c-sharp/concepts/strings/strings.md

```diff
@@ -0,0 +1,190 @@
+
+## Declaring and Initializing
+
+There are many different ways to declare and initialize a string in C#. The following example lists a number of them:
+
+```cs
+// Declare without initializing.
+string myString1;
+
+// Declare using System.String.
+System.String myString2;
+
+// Initialize with a string literal.
+string myString3 = "Hello World!";
+
+// Local variables can use implicit typing.
+var localString = "Still a strongly-typed string.";
+
+// The string constructor can create a string from a char*, char[] or sbyte*.
+char source = {'1','2','3','A','B','C'};
+string myString4 = new string(source);
+```
+
+> **Note:** The `new` operator is only used to create a string when initializing with an array of `char`.
+>
+> **Note:** It is best practice to initialize an empty string with the `System.String.Empty` constant value or the `""` string literal rather than with `null` to avoid an unwanted `NullRefrenceException`.
+
+## String Literals
+
+C# has three different types of string literals.
+
+### Quoted String Literals
+
+Quoted string literals are contained within a pair of double quote characters (`"`) on a single line. They require escape characters to include special characters and line breaks.
+
+```cs
+string myString1 = "I'm a quoted string literal.";
+
+string myString2 = "I have an embedded\r\nline break.";
+/* The resulting string:
+I have an embedded
+line break.
+*/
+```
+
+These are some of the escape sequences used in string literals:
+
+| Escape Sequence | Character Name                                               |
+| --------------- | ------------------------------------------------------------ |
+| `\'`            | Single quote                                                 |
+| `\"`            | Double quote                                                 |
+| `\\`            | Backslash                                                    |
+| `\0`            | Null                                                         |
+| `\n`            | New line                                                     |
+| `\r`            | Carriage return                                              |
+| `\t`            | Horizontal tab                                               |
+| `\uNNNN`        | UTF-16 Unicode character (range `\u0000` - `\uFFFF`)         |
+| `\U00NNNNNN`    | UTF-32 Unicode character (range `\U00000000` - `\U0010FFFF`) |
+
+### Verbatim String Literals
+
+Verbatim string literals are a convenience used for multi-line strings, strings that contain backslashes or embedded double quotes. Verbatim string literals begin with `@"` rather than just `"` as with a quoted literal string.
```

Comment:
```suggestion
Verbatim string literals are a convenience used for multi-line strings, strings that contain backslashes, or embedded double quotes. Verbatim string literals begin with `@"` rather than just `"` as with a quoted literal string.
```